libs/blocks/gnav/gnav.css

```diff
@@ -72,7 +72,7 @@ header nav.gnav::before {
   left: 0;
   right: 0;
   height: 56px;
-  background: #FFF;
+  background: var(--color-white);
```

Comment:
Very nice - boy scout rule!

Reply:
re colors - I agree with @vhargrave that most (all?) colors should be assigned to a descriptive var.

I do wonder about the basic colors though, would this be better as just `background: white;`?
Is there ever a time that `--color-white` would be changed to another color?

Nothing needs to be changed, just my musings on css color vars

Reply:
Agreed. @ryanmparrish and myself discussed this too. 
Maybe a basic plan around this would help moving forward.